test/functional/change_stream.test.js

```diff
@@ -2799,3 +2799,49 @@ describe('Change Stream Resume Error Tests', function() {
     })
   });
 });
+context('NODE-2626', function() {
```

Comment:
Maybe add the name of the ticket just to make this a bit more descriptive

Reply:
Sounds good. The name of the ticket was not very descriptive of the actual issue, so I'll use the name of the PR.